tools/marvin/marvin/config/test_data.py

```diff
@@ -513,6 +565,19 @@
             "NetworkACL": 'VpcVirtualRouter'
         }
     },
+    "vpc_offering_configdrive": {
+        "name": 'VPC offering ConfigDrive',
+        "displaytext": 'VPC offering ConfigDrive',
+        "supportedservices": 'Dhcp,StaticNat,SourceNat,NetworkACL,UserData,Dns',
+        "serviceProviderList": {
+            "Dhcp": "VpcVirtualRouter",
+            "StaticNat": "VpcVirtualRouter",
+            "SourceNat": "VpcVirtualRouter",
+            "NetworkACL": "VpcVirtualRouter",
+            "UserData": "ConfigDrive",
+            "Dns": "VpcVirtualRouter"
+        }
+    },
```

Comment:
@fmaximus can you move the test data to test itself? (for all the changes in test_data.py)

Reply:
@rhtyd are you saying that even for new generic component tests that are added (see test_configdrive.py being added), the policy is to add offering to the test itself rather than extending test_data , even when config drive becomes a new generic capability, added by this PR ?

Reply:
Yes, the test dsta should not be part of marvin library at all. We should stop doing that at least with new tests.